modules/cloud_logs/variables.tf

```diff
@@ -0,0 +1,132 @@
+variable "region" {
+  description = "The region where observability resources are created."
+  type        = string
+  default     = "eu-es"
+}
+
+variable "resource_group_id" {
+  type        = string
+  description = "The id of the IBM Cloud resource group where the instance(s) will be created."
+  default     = null
+}
+
+variable "cloud_logs_provision" {
```

Comment:
Whats the point in having this variable in the submodule? If your not going to provision an instance, then your just not going to call this module